channels/us.m3u

```diff
@@ -877,8 +877,8 @@ https://stream.swagit.com/live-edge/glendaleaz/smil:std-4x3-1-a/chunklist.m3u8
 https://dai.google.com/linear/hls/event/ChWV1GupQOWE92uG4DvbkQ/master.m3u8
 #EXTINF:-1 tvg-id="GlobalFashionChannel.us" tvg-name="Global Fashion Channel" tvg-country="US" tvg-language="English" tvg-logo="https://globalfashionchannel.com/wp-content/uploads/2016/03/GFC400r.png" group-title="Lifestyle",Global Fashion Channel
 https://vcngfcssai.teleosmedia.com/linear/globalfashionchannel/globalfashionchannel/playlist.m3u8
-#EXTINF:-1 tvg-id="GlobalGotTalentXUMO.us" tvg-name="Global Got Talent (XUMO)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/QHiOf7Z.png" group-title="",Global Got Talent (XUMO)
-https://dai2.xumo.com/xumocdn/p=redbox&deviceid=&is_lat=&subp=RedboxdesktopWebLinux/amagi_hls_data_xumo1212A-redboxgottalentglobal/CDN/playlist.m3u8
+#EXTINF:-1 tvg-id="GlobalGotTalentXUMO.us" tvg-name="Global Got Talent (XUMO)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/QHiOf7Z.png" group-title="Entertainment",Global Got Talent (XUMO)
+https://live-content.xumo.com/164/content/XM0PRM223TNM3Z/21324142/4.m3u8
```

Comment:
Isn't broadcast, returns a fixed chunk that ends

Reply:
Works!